src/accountsdb/db.zig

```diff
@@ -88,7 +88,7 @@ pub const AccountsDB = struct {
 
     /// track per-slot for purge/flush
     account_cache: RwMux(AccountCache),
-    file_map: RwMux(FileMap),
+    file_map_safe: ThreadSafeFileMap = .{},
```

Comment:
can probs come up with a better name than this, but this is what i have for now -- maybe `accountsbd.file_map.inner` is more accurate and we abstract most of it åway with methods?